src/tasks/deploy-utils.js

```diff
@@ -68,7 +68,17 @@ function validateId(settings) {
  */
 
 gulp.task('validate:id', () => {
-  const file = fs.readFileSync(config.tkConfig, 'utf8');
+  let file;
```

Comment:
Could probably be part of a future clean-up PR. Anything obvious stand out for you here?